### ISSUE TITLE
hello,Garethp!

### ISSUE THREAD

**Issue by synlol** [Garethp/php-ews#98]
```php
public function available_items()
    {
        $client = wm_exchange_logon();

        $request = array(
            'Items' => array(
                'GetUserAvailabilityRequest' => array(
                    'TimeZone'=>array(
                        'Bias'=>480,
                        'StandardTime'=>array(
                            'Bias'=>0,
                            'Time'=>'02:00:00',
                            'DayOrder'=>5,
                            'Month'=>10,
                            'DayOfWeek'=>'Sunday',
                        ),
                        'DaylightTime'=>array(
                            'Bias'=>-60,
                            'Time'=>'02:12300:00',
                            'DayOrder'=>1,
                            'Month'=>4,
                            'DayOfWeek'=>'Sunday',
                        )
                    ),
                    "MailboxDataArray"=>array(

                        "MailboxData" => array(
                            "Email"=> array(
                                "Address" => '563951553@qq.com'
                            ),
                            "AttendeeType" => 'Required',
                            "ExcludeConflicts" => false
                        )
                    ),
                    'FreeBusyViewOptions'=>array(
                        'TimeWindow'=>array(
                            'StartTime'=>'2017-6-9T00:00:00',
                            'EndTime'=>'2017-6-10T00:00:00',
                        ),
                        'MergedFreeBusyIntervalInMinutes'=>60,
                        'RequestedView'=>'DetailedMerged',
                    ),

                ),

            ),
        );

        //$request=NULL;

        $request = Message\GetUserAvailabilityRequest::buildFromArray($request);
        //print_r($request);exit;
        $response = $client->getClient()->GetUserAvailability($request);

        return $response;
    }
```

**Comment by synlol** [comment]
When I use the code above, "MailboxDataArray" without this property
When I use the code above, "MailboxDataArray" without this property
当我用上面的代码，“mailboxdataarray”没有这个属性

**Comment by Garethp** [comment]
Sorry, would you mind trying this on the new version I just released, `0.9.4`?
